provisioning/service/src/AttestationMechanism.cs

```diff
@@ -36,10 +36,10 @@ public AttestationMechanism()
         [JsonProperty(PropertyName = "tpm")]
         public TpmAttestation Tpm
         {
-            get { return this.tpm; }
+            get { return this._tpm; }
```

Comment:
We should remove the `this.` part. "_" means that it's a local property already.